internal/tracinginterceptor/interceptor.go

```diff
@@ -90,3 +148,63 @@ func (m *Interceptor) CallStream(ctx context.Context, req *transport.StreamReque
 	// TODO: implement
 	panic("implement me")
 }
+
+func updateSpanWithError(span opentracing.Span, err error) error {
+	if err == nil {
+		return err
+	}
+
+	ext.Error.Set(span, true)
+	if status := yarpcerrors.FromError(err); status != nil {
+		errCode := status.Code()
+		span.SetTag("rpc.yarpc.status_code", errCode.String())
+		span.SetTag("error.type", errCode.String())
+		return err
+	}
+
+	span.SetTag("error.type", "unknown_internal_yarpc")
+	return err
+}
+
+func updateSpanWithResponseError(span opentracing.Span, res *transport.Response, err error) error {
+	if err == nil && (res == nil || !res.ApplicationError) {
+		return err
+	}
+
+	ext.Error.Set(span, true)
+	status := yarpcerrors.FromError(err)
+	if status != nil {
+		errCode := status.Code()
+		span.SetTag("rpc.yarpc.status_code", errCode.String())
+		span.SetTag("error.type", errCode.String())
+		return err
+	}
+
+	if res != nil && res.ApplicationError {
+		span.SetTag("error.type", "application_error")
+		return err
+	}
+
+	span.SetTag("error.type", "unknown_internal_yarpc")
+	return err
+}
+
+// GetPropagationFormat returns the opentracing propagation depends on transport.
+// For TChannel, the format is opentracing.TextMap
+// For HTTP and gRPC, the format is opentracing.HTTPHeaders
+func GetPropagationFormat(transport string) opentracing.BuiltinFormat {
```

Comment:
Nit: since we move those propagation related funcs to  the same package, there is no need to make them exported.